ps_mainmenu.php

```diff
@@ -1026,143 +1034,144 @@ public function renderForm()
+            $fields_form = [
+                'form' => [
+                    'legend' => [
+                        'title' => $this->trans('Menu Top Link', [], 'Modules.Mainmenu.Admin'),
+                        'icon' => 'icon-link',
+                    ],
+                    'input' => [
+                        [
                             'type' => 'link_choice',
                             'label' => '',
                             'name' => 'link',
                             'lang' => true,
-                        )
-                    ),
-                    'submit' => array(
+                        ],
+                    ],
+                    'submit' => [
                         'name' => 'submitBlocktopmenu',
-                        'title' => $this->trans('Save', array(), 'Admin.Actions')
-                    )
-                ),
-            );
+                        'title' => $this->trans('Save', [], 'Admin.Actions'),
+                    ],
+                ],
+            ];
         } else {
-            $fields_form = array(
-                'form' => array(
-                    'legend' => array(
-                        'title' => $this->trans('Menu Top Link', array(), 'Modules.Mainmenu.Admin'),
-                        'icon' => 'icon-link'
-                    ),
-                    'info' => '<div class="alert alert-warning">'.
-                        $this->trans('All active products combinations quantities will be changed', array(), 'Modules.Mainmenu.Admin').'</div>',
-                    'submit' => array(
+            $fields_form = [
+                'form' => [
+                    'legend' => [
+                        'title' => $this->trans('Menu Top Link', [], 'Modules.Mainmenu.Admin'),
+                        'icon' => 'icon-link',
+                    ],
+                    'info' => '<div class="alert alert-warning">' .
+                        $this->trans('All active products combinations quantities will be changed', [], 'Modules.Mainmenu.Admin') . '</div>',
+                    'submit' => [
                         'name' => 'submitBlocktopmenu',
-                        'title' => $this->trans('Save', array(), 'Admin.Actions')
-                    )
-                ),
-            );
+                        'title' => $this->trans('Save', [], 'Admin.Actions'),
+                    ],
+                ],
+            ];
         }
 
         $helper = new HelperForm();
         $helper->show_toolbar = false;
         $helper->table = $this->table;
-        $lang = new Language((int)Configuration::get('PS_LANG_DEFAULT'));
+        $lang = new Language((int) Configuration::get('PS_LANG_DEFAULT'));
         $helper->default_form_language = $lang->id;
         $helper->allow_employee_form_lang = Configuration::get('PS_BO_ALLOW_EMPLOYEE_FORM_LANG') ? Configuration::get('PS_BO_ALLOW_EMPLOYEE_FORM_LANG') : 0;
-        $this->fields_form = array();
+        $helper->fields_form = [];
```

Comment:
The module is not ready 

Check https://github.com/PrestaShop/PrestaShop/issues/25142